pkg/clients/fivetran/team_membership.go

```diff
@@ -56,6 +58,64 @@ func (fc *FivetranClient) FetchTeamMembersByTeamID(
 
 }
 
+// starting from here to faalback
+func (fc *FivetranClient) AddUserstoTeam(ctx context.Context, teamID string, userIDs []string) error {
+
+	sem := make(chan struct{}, 5)
+	errr := make(chan error, len(userIDs))
+	var w sync.WaitGroup
+	for _, userID := range userIDs {
+		w.Add(1)
```

Comment:
instead of defining extra functions, can we just update the existing function `AddUserToTeam` as this contains the implementation logic for fivetran.